src/rvsdg/from_cfg.rs

```diff
@@ -75,12 +79,15 @@ pub(crate) fn cfg_func_to_rvsdg(cfg: &mut Cfg) -> Result<RvsdgFunction> {
     })
 }
 
+pub(crate) type FunctionTypes = HashMap<String, Option<Type>>;
```

Comment:
Why Option<Type> (and document it here)

Reply:
Left a comment- functions don't always return a value so for these we use `None`. Bril doesn't have a void type